ipu.py

```diff
@@ -228,11 +231,16 @@ def __init__(self, full_url: str, user: str = "root", password: str = "calvin"):
             password = "calvincalvincalvin"
         super().__init__(full_url, user, password)
 
-    def _run_curl(self, command: str) -> host.Result:
+    def _run_curl(self, command: str, *, quiet: bool = False) -> host.Result:
         lh = host.LocalHost()
-        logger.info(command)
-        result = lh.run(f"curl -v -u {self.user}:{self.password} {command}")
-        logger.info(result)
+        if not quiet:
+            logger.info(command)
+        result = lh.run(
+            f"curl -v -u {self.user}:{self.password} {command}",
```

Comment:
time to refactor to use requests instead.

Reply:
why does this not use `ailib.Redfish` API (or, if the suitable API is not there, contribute it there first)?

Reply:
Remove unnecessary line breaks. Then we can merge this.